test/smoke/testApps/SpringBootAuto/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/SpringBootAutoTest.java

```diff
@@ -1,13 +1,67 @@
 package com.microsoft.applicationinsights.smoketest;
 
-import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import com.google.common.base.Stopwatch;
+import com.microsoft.applicationinsights.internal.schemav2.Data;
+import com.microsoft.applicationinsights.internal.schemav2.Envelope;
+import com.microsoft.applicationinsights.internal.schemav2.RemoteDependencyData;
+import com.microsoft.applicationinsights.internal.schemav2.RequestData;
+import org.junit.*;
+
+import static org.junit.Assert.*;
 
 @UseAgent
 public class SpringBootAutoTest extends AiSmokeTest {
 
     @Test
     @TargetUri("/test")
     public void doMostBasicTest() throws Exception {
-        mockedIngestion.waitForItems("RequestData", 1);
+        List<Envelope> rdList = mockedIngestion.getItemsEnvelopeDataType("RequestData");
+        boolean found = false;
+        for (Envelope envelope : rdList) {
+            RequestData rd = (RequestData) ((Data) envelope.getData()).getBaseData();
+            if (rd.getName().equals("GET /test")) {
+                found = true;
+            }
+        }
+
+        assertTrue(found);
+    }
```

Comment:
is this test needed?

Reply:
it didn't do anything before.. i just want it to pass with this change.